spec/features/admin_requests_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+describe 'Viewing all requests' do
+  describe 'by a webauth user' do
+    let(:current_user) { create(:superadmin_user) }
```

Comment:
In the context of a controller (controller/feature specs) you'll need to stub the current_user method on the controller.  I've added a helper method to do this `stub_current_user`.

By default it will create an anonymous user, but takes an optional parameter for a user object itself.

In your before block you can call 
```ruby
stub_current_user(create(:superadmin_user))
```
and this will create a `superadmin` user and stub the `current_user` method on the controller with that user object.